docs/_articles/en/v3.x/code-analyzer-vs-code-extension.md

```diff
@@ -50,13 +50,13 @@ Complete one of these options to perform a Code Analyzer (beta) scan.
 * To scan selected files or folders:
 
 1. Select a group of files or folders.
-2. Right click in the VS Code Explorer and select **SFDX: Scan selected files or folders with Code Analyzer**.
+2. Right click in the VS Code Explorer and select **SF: Scan selected files or folders with Code Analyzer**.
 
 * To scan a single code file:
 
 1. Open a code file in the VS Code Editor.
-2. From the VS Code Command Palette, choose **SFDX: Scan current file with Code Analyzer**.
-3. Alternatively, right click in the VS Code Editor and select **SFDX: Scan current file with Code Analyzer**.
+2. From the VS Code Command Palette, choose **SF: Scan current file with Code Analyzer**.
+3. Alternatively, right click in the VS Code Editor and select **SF: Scan current file with Code Analyzer**.
```

Comment:
For lines 58 and 59, see the comment at line 53.

Reply:
Ignoring per comment thread on line 53 @jfeingold35